setup.py

```diff
@@ -8,22 +8,44 @@
 '''
 from setuptools import setup, find_packages
 
-import voxel51.constants as voxc
-
 
+# @note version info should be kept in-sync with `voxel51/version.json`
```

Comment:
Why not just get this information from `voxel51/version.json` directly?

Reply:
Just for consistency with how we're now doing it in other libraries.

Is it possible to access this `setup` information programmatically after the package is installed? In that case, I'd like to delete `version.json` and have this file be the source of truth for that information.

Reply:
There's a library in Python 3.8 which apparently does this: https://docs.python.org/3.8/library/importlib.metadata.html, which has been backported here: https://importlib-metadata.readthedocs.io/en/latest/